content/kb/troubleshooting-rwx-mount-ownership-set-to-nobody.md

```diff
@@ -0,0 +1,85 @@
+---
+title: "Troubleshooting: Longhorn RWX shared mount ownership is shown as nobody in consumer Pod"
+author: Chin-Ya Huang
+draft: false
+date: 2021-03-31
+categories:
+  - "rwx"
+---
+
+## Applicable versions
+
+All Longhorn versions.
+
+## Symptoms
+
+When Pod mounts with RWX volume, the Pod share mount directory and all of the ownership of its recurring contents are shown as nobody, and in the share-manager is shown as root.
```

Comment:
, but in the share manager is shown as root.

Reply:
Fixed